core/src/main/java/com/arcrobotics/ftclib/util/FTCLibOpMode.java

```diff
@@ -0,0 +1,45 @@
+package com.arcrobotics.ftclib.util;
+
+import com.arcrobotics.ftclib.gamepad.GamepadEx;
+import com.qualcomm.hardware.lynx.LynxModule;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+
+import java.util.List;
+
+public abstract class FTCLibOpMode extends LinearOpMode {
+
+    protected GamepadEx driverOp, toolOp;
+
+    @Override
+    public final void runOpMode() throws InterruptedException {
+        driverOp = new GamepadEx(gamepad1);
+        toolOp = new GamepadEx(gamepad2);
+
+        initialize();
+
+        List<LynxModule> hubs = hardwareMap.getAll(LynxModule.class);
+        hubs.forEach(hub -> hub.setBulkCachingMode(LynxModule.BulkCachingMode.MANUAL));
+
+        waitForStart();
+
+        while (!isStopRequested() && opModeIsActive()) {
+            hubs.forEach(LynxModule::clearBulkCache);
```

Comment:
Make sure to document this! Otherwise people are going to run `clearBulkCache` themselves.